.github/workflows/validate_linux_rt.yml

```diff
@@ -0,0 +1,34 @@
+name: Validate Linux RT Feed
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
Wont this also happen when a PR is tagged with breaking change or if we used tags for other things?

Reply:
This should only trigger on tags which match the filter `v*.*.*` are pushed. It would trigger if you used that format of tags for other things but if tags are only used when creating a new release this should be fine.

Reply:
Yeah, also in the recent update I changed the filter to 'v[0-9]+.[0-9]+.[0-9]+' to restrict it a bit more. Realized we probably only want to trigger this check for release tags pushed up and not pre-releases (v1.5.1-rc0 or v1.5.1-internal).